### ISSUE TITLE
Enable Visibility based on certain conditions

### ISSUE THREAD

**Issue by lmcintyre** [SheepGoMeh/VisibilityPlugin#7]
Visibility is most useful to me in incredibly populated areas. A system could be added to take into account various things, including, but not limited to:
- Current territory
- Number of players in actor table
- Current job
and other things of that nature. How customizable you want it is up to you. Territory-specific setting is what I came here to submit but got carried away...

**Comment by skifty** [comment]
Yeah I'd love an option just to enable the plugin in major cities, or a toggle command at least.

**Comment by skullriot** [comment]
Second the suggestion to hide based on class. I'd love to be able to hide all crafters causing such a rukus. Not sure if it would be possible to only hide crafters that are actively crafting or not, and leave the ones that aren't doing anything visible.

**Comment by Nysterian** [comment]
Personally, alliance raids are very laggy for me, even though I'm already hiding everyone's effects. So I'd like to hide every member that isn't in my party, or dead

**Comment by SheepGoMeh** [comment]
The plugin will never be enabled in duties as it can interfere with whatever duty mechanic that might involve multiple persons outside of your party.

**Comment by Nicolas-GE** [comment]
i would love it to be able to turn on based on fps, so if the fps goes under a set minimum, it would enable the plugin untill you leave the area or for a set amount of time, this would be useful for hunts where the game becomes quite unplayable with how many players are attending them
